src/TRestAxionField.cxx

```diff
@@ -94,9 +88,10 @@ void TRestAxionField::Initialize() {
 /// The result will be given for an axion-photon coupling of 10^{-10} GeV^{-1}
 ///
 double TRestAxionField::BL(Double_t Bmag, Double_t Lcoh) {
-    Double_t lengthInMeters = Lcoh / 1000.;
+    Double_t lengthInMeters = Lcoh * units("m");
```

Comment:
What does `units("m")` do? If its purpose is to convert a length in m to natural units then we need to first have Lcoh in m rather than mm. Maybe I have misunderstood this so let me know.

Reply:
It's a conversion factor from the standard REST units (mm for length)  to meters, that's what I understand